<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="question_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="answer_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>580</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>600</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="choiceBackground_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hp_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(100, 50, 0);</string>
    </property>
    <property name="text">
     <string>Score  :  </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="easteregg">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>25</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(160, 80, 0);</string>
    </property>
    <property name="text">
     <string>'^'</string>
    </property>
   </widget>
   <zorder>choiceBackground_label</zorder>
   <zorder>question_label</zorder>
   <zorder>answer_btn</zorder>
   <zorder>answer_lineEdit</zorder>
   <zorder>hp_label</zorder>
   <zorder>score_label</zorder>
   <zorder>score</zorder>
   <zorder>easteregg</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
